map_gui/src/tools/city_picker.rs

```diff
@@ -327,7 +327,7 @@ impl<A: AppLike + 'static> AllCityPicker<A> {
                     ctx.style().btn_close_widget(ctx),
                 ]),
                 Widget::row(vec![
-                    Widget::draw_svg(ctx, "system/assets/tools/search.svg"),
+                    Image::icon("system/assets/tools/search.svg").into_widget(ctx),
```

Comment:
A little more verbose in the simplest case, but with the benefit of flexibility!

If you prefer to keep a more terse constructor around, I could add something like:
`Image::icon_widget("system/assets/tools/search.svg", ctx)`

Reply:
I prefer the explicit builder style, thanks for setting it up! `.tooltip()` is a nice touch; collapses all those weird one-off `Widget` methods.